recipes-rubygems/rubygems-ffi_1.15.1.bb

```diff
@@ -15,8 +15,8 @@ GEM_INSTALL_FLAGS += "\
     --with-opt-dir=${RECIPE_SYSROOT} \
 "
 
-SRC_URI[md5sum] = "d5d910a1a5b079caf88d73bd2c956a3c"
-SRC_URI[sha256sum] = "621057e9bd2bd5771a072015ed57b71f9be43b5d9f18b53863d4bbb5c772015b"
+SRC_URI[md5sum] = "63f782c2c7dc4db127da72b5bcec53c6"
+SRC_URI[sha256sum] = "d125ee6a8b88405926f8af9b3d25a75ae93e1c782cd50bd62e81e20cce16d696"
 
 GEM_NAME = "ffi"
```

Comment:
potential issue: 'EXTRA_RDEPENDS' is unknown, maybe you mean 'EXTRA_IMAGEDEPENDS'